examples/flutter_gallery/pubspec.yaml

```diff
@@ -6,6 +6,7 @@ dependencies:
 
   flutter:
     sdk: flutter
+  url_launcher: ^0.3.1
```

Comment:
Why leave out the version?  We might want to create a new version of url_launcher in the future without breaking old versions of the gallery.